packages/docusaurus-theme-classic/src/theme/Tabs/index.js

```diff
@@ -17,8 +18,27 @@ const keys = {
 };
 
 function Tabs(props) {
-  const {block, children, defaultValue, values} = props;
+  const {block, children, defaultValue, values, groupId} = props;
+  const {getTabGroupChoice, setTabGroupChoice} = useTabGroupChoiceContext();
   const [selectedValue, setSelectedValue] = useState(defaultValue);
+
+  if (typeof groupId !== 'undefined') {
```

Comment:
It seems to me that we can remove the `groupId` prop (this is somewhat redundant) in favor of `value` prop in `TabItem` component to identify each tab? Yes, this way we synchronize tabs by default and do not give users prevent this behavior. (Although I do not think that is a big problem).